services/storage/tests/conftest.py

```diff
@@ -295,18 +295,17 @@ def dsm_mockup_db(postgres_service_url, s3_client, mock_files_factory):
     dcw = DatcoreWrapper(api_token, api_secret, loop, pool)
 
     await dcw.create_test_dataset(BUCKET_NAME)
-
     tmp_files = mock_files_factory(2)
     for f in tmp_files:
         await dcw.upload_file(BUCKET_NAME, os.path.normpath(f))
 
-    ready = False
-    counter = 0
-    while not ready and counter < 5:
-        data = await dcw.list_files()
-        ready = len(data) == 2
-        await asyncio.sleep(10)
-        counter = counter + 1
+    # ready = False
```

Comment:
pls try not to leave commented code without explanations. Think that this code should be readable by others